code/modules/hydroponics/hydro_tray.dm

```diff
@@ -619,6 +619,59 @@
 	if(..())
 		return 1
 
+	if(istype(O, /obj/item/claypot))
+		user << "<span class='warning'>You must place the pot on the ground and use a spade on \the [src] to make a transplant.</span>"
+		return
+
+	if(seed && istype(O, /obj/item/weapon/pickaxe/shovel/spade))
```

Comment:
Is that supposed to be rhetorical because the answer is a qualified yes, I use a shovel every weekend when I take care of my yard to fill planters with soil or dig up plants or work clay into mounds for better irrigation.